packages/toast/src/Toast.story.tsx

```diff
@@ -127,7 +128,14 @@ export const Variants: ComponentStory<typeof InternalToast> = (
                 pushToast({
                   ...props,
                   title: `I'm a ${variant} toast`,
-                  description: faker.lorem.lines(random(1)),
+                  description: (
+                    <>
+                      {randomText}&nbsp;
+                      {randomText && (
+                        <Link href="http://localhost:9001">Link style</Link>
+                      )}
```

Comment:
When there's no `randomText` this still returns a `nbsp;` which makes the toast render weird 
<img width="434" alt="Screenshot 2023-04-21 at 2 54 12 PM" src="https://user-images.githubusercontent.com/2414030/233713478-619fb473-1a02-40af-89c6-c67924ef0ef3.png">